tests/test_pint_units.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from pnpq.units import ureg
+
+
+def test_mpc320_step_conversion() -> None:
+
+    for value in [-100, 0, 100]:  # Test few values
```

Comment:
This is a good opportunity to use a parameterized test:

https://docs.pytest.org/en/stable/how-to/parametrize.html

Reply:
Thank you for the idea. I will get to it.